services/virtualmachine/call_system.go

```diff
@@ -23,7 +23,8 @@ func (s *service) callSystemContract(
 ) (protocol.ExecutionResult, *protocol.ArgumentArray, error) {
 
 	// create execution context
-	executionContextId, executionContext := s.contexts.allocateExecutionContext(blockHeight, blockHeight, blockTimestamp, protocol.ACCESS_SCOPE_READ_ONLY, nil)
+	// currentBlockProposerAddress is nil because in call system it is meaningless.
```

Comment:
why is it meaningless?

Reply:
because the only use currently for this is getOrderedCommittee which doesn't do any calculation based on proposer.